adoc/Private-Registry-Administration.adoc

```diff
@@ -110,35 +110,251 @@ $ uaac client add suse_private_registry -s suseprivateregistrysupersecret --scop
 . Uncheck *UAA Verify Cert* if the UAA server uses a self-signed or untrusted certificate.
 . Click *Save* to complete the configuration.
 
-=== Registry Deployment Configuration Changes
+=== Registry Helm Configuration Changes
```

Comment:
Hm, this looks bit weird. I think _Registry Deployment Configuration Changes_ made sense, _Registry Helm Configuration Changes_ not that much ... I do not think that when you talk about configuration changes, you do not need to mention the deployment _method_ in the title.

Maybe just use _Registry Configuration Changes_.
Or _Post-deployment Changes_

Reply:
OK